### ISSUE TITLE
Alt text is reversed for layer hide/show button

### ISSUE THREAD

**Issue by rmx90210** [maplibre/maputnik#1675]
**Maputnik version**: 3.0.0
**Browser**: Brave
**OS**: macOS

**Description of the bug**:

Alt text is reversed when you hover over the show/hide toggle for a layer.

**Steps to reproduce the behavior**:
1. Add a layer
2. Hover over the show/hide button (👁️)
3. When the layer is hidden, you'll see the alt text "hide" when the action verb should be "show", and vice versa.

**Style file or style URL**:

n/a

**Screenshots**:

<img width="238" height="73" alt="Image" src="https://github.com/user-attachments/assets/d1494626-e7f1-4b2e-8d18-9a193d7a8886" />

<img width="243" height="75" alt="Image" src="https://github.com/user-attachments/assets/14be1a61-7fb1-448a-acc3-5d1ae2fa3d77" />

**Comment by HarelM** [comment]
Feel free to submit a PR to fix it, should be fairly easy I believe...
